documentation/getting-started.adoc

```diff
@@ -0,0 +1,245 @@
+== Getting started
+
+=== Prerequisites
+
+Kubernetes or OpenShift cluster are required to deploy Strimzi. Strimzi supports all kind of clusters - from public and
```

Comment:
A Kubernetes or OpenShift cluster is required...

Strimzi supports all kinds of cluster